README.md

```diff
@@ -17,23 +29,48 @@ Once every new shell, source the setup script to initilaize the enviornment.
 
     source setup.sh
 
-
-
 Compile the code with option flags
 
-    sdl_make_tracklooper -m8
+    sdl_make_tracklooper -mc8
     -c: run with the cmssw caching allocator
     -h: show help screen with all options
- Run the code
+
+Run the code
  
-    ./bin/sdl -n <nevents> -v <verbose> -w <writeout> -s <streams> -i <dataset>
+    sdl -n <nevents> -v <verbose> -w <writeout> -s <streams> -i <dataset> -o <output>
+
     -i: PU200; muonGun, etc
     -n: number of events
     -v: 0-no printout; 1- timing printout only; 2- multiplicity printout
     -s: number of streams/events in flight
-    -w: 0- no writout; 2- full ntuple writeout
+    -w: 0- no writout; 1- minimum writeout; 2- full ntuple writeout
+    -o: provide an output root file name (e.g. LSTNtuple.root)
+    
+Plotting numerators and denominators of performance plots
+
+    createPerfNumDenHists -i <input> -o <output> [-g <pdgids> -n <nevents>]
+
+    -i: Path to LSTNtuple.root
+    -o: provide an output root file name (e.g. num_den_hist.root)
+    -n: (optional) number of events
+    -g: (optional) comma separated pdgids to add more efficiency plots with different sim particle slices
     
+Plotting performance plots
+
+    lst_plot_performance.py num_den_hist.root -t "mywork"
+
+When running on ```cgpu-1``` remember to specify python3 as there is no python.
+The shebang on the ```lst_plot_performance.py``` is not updated as ```lnx7188``` works with python2....
```

Comment:
I checked and, even though `python3` works on `lnx7188`, the command `python3 efficiency/python/lst_plot_performance.py LSTNumDen.root -t "myTag"` doesn't.

I checked the same thing on `phi3` the latter command again doesn't work, even though the former does.

The error is the following:
```
Traceback (most recent call last):
  File "efficiency/python/lst_plot_performance.py", line 4, in <module>
    import ROOT as r
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc900/lcg/root/6.20.06-ghbfee3/lib/ROOT.py", line 24, in <module>
    import cppyy
  File "/cvmfs/cms.cern.ch/slc7_amd64_gcc900/lcg/root/6.20.06-ghbfee3/lib/cppyy.py", line 61, in <module>
    import libPyROOT as _backend
ImportError: dynamic module does not define module export function (PyInit_libPyROOT)
```

Therefore, I suspect that this is an issue with the (older) version of ROOT we are using with our `setup.sh` on those machines not supporting `python3` rather than an actual issue with `python3`.

Reply:
I think this is some pesky CMSSW stuff. Probably that ROOT libraries are only available for python2 in the older versions (CMSSW 11), and only in python3 in the newer (CMSSW 12) versions